src/map.rs

```diff
@@ -44,6 +46,18 @@ impl Map {
         self.layers.insert(layer_id.into(), layer_entity);
     }
 
+    /// Initialize new Layer Builder for the Map
+    pub fn layer_builder<T: TileBundleTrait>(
+        &mut self,
+        commands: &mut Commands,
+        settings: &LayerSettings,
+        layer_id: impl LayerId,
+    ) -> LayerBuilder<T> {
+        let builder = LayerBuilder::new(commands, settings.clone(), self.id, layer_id);
+        self.add_layer(commands, layer_id, builder.get_entity());
+        builder
+    }
```

Comment:
this makes it easier and less verbose to get LayerBuilder for map (doesn't require passing map_id). Old constructor is still available though.